examples/plugins/writing-plugins.rst

```diff
@@ -0,0 +1,131 @@
+Writing Your Own MLflow Plugin(s)
+---------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+MLflow Python API plugins provide a powerful mechanism for customizing the behavior of the MLflow
+Python client, allowing you to define custom behaviors for logging metrics, params, and artifacts,
+set special context tags at run creation, and override model registry methods for registering
+models etc.
```

Comment:
Nit: `etc.` sounds a bit odd when used in conjunction with `and`. Maybe use one or the other?